### PR TITLE
Rl 158  - Displaying suffixes

### PR DESCRIPTION
**Pull request by mlipardo**
### Jira Story

- [Suffixes are missing on the relationship card RL-158](https://osi-cwds.atlassian.net/browse/RL-158)

## Tests
- [x] I have included unit tests 
- [x] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

- Display First, Middle, Last and Suffix of person on header and table on the relationship card edit and create cards
- This will also impact snapshot.

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: spec/features/screening/relationships_card_spec.rb [hunk 1]
```diff
@@ -341,6 +361,7 @@
               expect(page).to have_content('Jane Campbell')
               expect(page).to have_content('Jake Campbell Brother (Half)')
               expect(page).to have_content('05/05/1990 (28 yrs)')
+              expect(page).to have_content('John Florence, PhD Brother (Half)')
```
[the remark this answers is not in the document]
Comment (mlipardo) [reply spec/features/screening/relationships_card_spec.rb]: Added a test expectation for DOB for the added relationship - John.

**Comment by mlipardo** [comment]
@wmitchell-cambria Thanks!

File: spec/javascripts/utils/nameFormatterSpec.js [hunk 2]
```diff
@@ -235,6 +282,36 @@ describe('nameFormatter', () => {
         name_suffix: undefined,
       })).toEqual('Bill S. Preston')
     })
+
+    it('returns the name with suffix for a suffix with special char.', () => {
+      expect(nameFormatter({
+        first_name: 'Bill',
+        middle_name: 'S.',
+        last_name: 'Preston',
+        name_suffix: 'jr.',
```
Comment (NagaPradeepGajula-osi) [inline spec/javascripts/utils/nameFormatterSpec.js]: nice coverage on suffix conversion specs. 👍

Comment (mlipardo) [reply spec/javascripts/utils/nameFormatterSpec.js]: Thanks!

**Pull request merged**